main.go

```diff
@@ -12,7 +11,7 @@ func main() {
 	if err != nil {
 		return
 	}
-	fmt.Println(s)
+	pp.Print(s)
```

Comment:
__[gometalinter]__
* [errcheck] error return value not checked (pp.Print(s))

<!-- Comment by [SideCI](https://sideci.com) -->